### ISSUE TITLE
Issues with sample documents in Neon - empty syllables and inconsistent correction

### ISSUE THREAD

**Issue by JoyfulGen** [DDMAL/Neon2#1012]
**Empty Syllables**

A while ago, a notification was added to Neon to tell the user when empty syllables (syllables containing no neumes, and therefore impossible to interact with) had been found in an MEI file. These empty syllables don't appear anymore in new folios, but they are present in the sample documents from Salzinnes. This means that opening a sample document triggers a whole bunch of these notifications. Behold:

https://github.com/DDMAL/Neon/assets/83373378/0d594485-0c84-43e2-a890-d2a9e6521750

 The "MEI Actions" dropdown menu allows the user to remove those empty syllables, but it's my understanding that those MEI actions should only be available to developers and testers, not to the general public. Since the sample documents are meant for new Neon users, I feel that it would be better to eliminate the empty syllables entirely. @yinanazhou, @annamorphism, is this sensible, and also possible?

For reference, here are now-closed GitHub issues where empty syllables were discussed (it was a bit of a saga):
https://github.com/DDMAL/Neon/issues/800
https://github.com/DDMAL/MEI_encoding/issues/29
https://github.com/DDMAL/Neon/issues/727

**Inconsistent correction**

The sample documents appear to have been only partly corrected, and I'm not sure whether this was intentional. All symbols are in the correct place (but some C clefs are wrong), staves are reasonably correctly aligned, as are bounding boxes. But neume and syllable groupings are completely wrong. @annamorphism, what are these sample files supposed to look like? Fully corrected, or fully uncorrected? As it stands, I feel that this halfway-there state is actually kind of confusing, because it looks at first sight like it's corrected, but it's really not.

Fully corrected Salzinnes files are in this repo: https://github.com/DDMAL/e2e-omr-resources/tree/main/resulting_mei_files/Salzinnes/reviewed. Uncorrected files are slightly harder to locate; I have several, but not all.

**Comment by annamorphism** [comment]
I think it's probably best if the sample documents are fully corrected and free of empty syllables.

**Comment by yinanazhou** [comment]
I agree that it's the best if the sample documents are fully corrected. I tried to update the sample MEI files, but they don't match with the images. I tried both 005r and 005v for CF-005, 009r and 009v for CF-009. None of them matches with the image.

**Comment by yinanazhou** [comment]
Hi @annamorphism and @JoyfulGen, I'm a bit confused about the sample file names. I checked with Cantus Ultimus. CF-005 in Neon sample files matches 002r in Cantus Ultimus Salzinnes. But the rendered MEI is a bit off with the source image in sample files. I think it's because the images are a little bit different in terms of scale and position.

Neon sample files with 002r MEI:
![image](https://github.com/DDMAL/Neon/assets/74484576/c98ca3a9-2917-4bad-9a79-902f0807c392)

Cantus Ultimus:
![image](https://github.com/DDMAL/Neon/assets/74484576/6e0d1675-aed8-4b8f-a579-0998a9ee1506)

How should I fix it in this case?

**Comment by annamorphism** [comment]
How mysterious! I'm guessing from the names that the images are peeled out of the manifest for the whole Salzinnes manuscript. But which one?  In the Leuven manifest 2r is the 9th image, and in the Montreal set it's 3--although it would be the 5th one if the pictures of the cover were included, which I remember them being in the whole-manuscript demo (which I can't access currently.) So maybe the MEI will line up with the older white-background Montreal images that are on 
Cantus Ultimus. The colours are a little different, though, which I hope is just some touch-up step and not an indicator of a third set of images altogether...

**Comment by yinanazhou** [comment]
@annamorphism in this case, what should I do about the sample documents? I can replace them with some files in https://github.com/DDMAL/e2e-omr-resources/tree/main/resulting_mei_files/Salzinnes/reviewed with images in one drive.

**Comment by yinanazhou** [comment]
Hi @annamorphism and @JoyfulGen, the sample files have already been updated. One thing I noticed is that 032r is missing from the reviewed folder.

**Comment by JoyfulGen** [comment]
Heya @yinanazhou and @annamorphism, I've looked at a few of these and they look so much better! Thanks Yinan :) I do have two questions:
- All files are showing an INVALID message, because of the wrong mei version thing. Does this matter? Will the average user even be able to see the VALID/INVALID messages? If yes, then this might be confusing.
- Some files (002r and 023r, for example) are also displaying `Missing the 2nd part of the toggle-linked syllable` messages. These half-toggled syllables don't occur anymore, but it would seem that they were still being created when we were correcting Salzines. Again, will the average user even see this message? If not, then this doesn't really matter, because there's the half-toggle doesn't actually change much, so it's impossible to know it's there without the error message. If yes, then this is also confusing. I could remove the half-toggles, if we want them gone.

**Comment by yinanazhou** [comment]
Right now the gh-pages version of neon does not have a viewer mode, so the average user can see everything. But we can definitely bring viewer mode back if needed.

**Comment by JoyfulGen** [comment]
Sample files look great now! Closing issue.
